bundles/org.openhab.binding.modbus.e3dc/src/test/java/org/openhab/binding/modbus/e3dc/util/DataConverterTest.java

```diff
@@ -30,17 +30,58 @@
 public class DataConverterTest {
 
     @Test
-    public void testE3DCSwapValueNegative() {
-        // Reg 69 value 65098 bytes [-2, 74]
-        // Reg 70 value 65535 bytes [-1, -1]
-        byte[] b = new byte[] { -2, -74, -1, -1 };
-        assertEquals(-330, DataConverter.getInt32Swap(ByteBuffer.wrap(b)), "Negative Value");
```

Comment:
This test not needed anymore?

Reply:
Correct. Method under test no longer exist. It was in fact a more convoluted implementation of a conversion that exists in transport module.

While this test has been added, you should be able to find the same "byte sequence", representing negative value, tested in a new test introduced in data block test.